vaadin-spring-tests/test-ts-form/frontend/views/form/vaadin-elements-view.ts

```diff
@@ -0,0 +1,113 @@
+import { LitElement, html} from 'lit';
+import { customElement, query } from 'lit-element';
+import {until} from 'lit-html/directives/until.js';
+import {repeat} from 'lit-html/directives/repeat.js';
+
+import '@vaadin/vaadin-form-layout';
+import '@vaadin/vaadin-checkbox';
+import '@vaadin/vaadin-checkbox/vaadin-checkbox-group';
+import '@vaadin/vaadin-combo-box';
+import '@vaadin/vaadin-date-picker';
+import '@vaadin/vaadin-date-time-picker';
+import '@vaadin/vaadin-time-picker';
+import '@vaadin/vaadin-select';
+import '@vaadin/vaadin-list-box';
+import '@vaadin/vaadin-item';
+import '@vaadin/vaadin-radio-button';
+import '@vaadin/vaadin-radio-button/vaadin-radio-group';
+import '@vaadin/vaadin-text-field';
+import '@vaadin/vaadin-text-field/vaadin-integer-field';
+import '@vaadin/vaadin-text-field/vaadin-number-field';
+import '@vaadin/vaadin-text-field/vaadin-email-field';
+import '@vaadin/vaadin-rich-text-editor';
+import '@vaadin/vaadin-login';
+import '@vaadin/vaadin-custom-field';
+import '@vaadin/vaadin-button';
+import '@vaadin/vaadin-notification';
+
+import { field, Binder } from '@vaadin/form';
+
+import ElementsModel from 'Frontend/generated/com/vaadin/flow/spring/form/data/ElementsModel';
+import { ElementsEndpoint } from 'Frontend/generated/ElementsEndpoint';
+
+@customElement('vaadin-elements-view')
+export class VaadinElementsView extends LitElement {
```

Comment:
Since this PR also includes the View and Layout classes from Vaadin Start this should probably extend View instead of LitElement directly. Then we can also leave out the `createRenderRoot()` override at the end. Or is there some reason to include those helpers from Vaadin Start but not use them?

Reply:
good point.
Also realized that we can go one step further, we can reuse the existing fusion security test module for form as well instead of creating yet another module.